### PR TITLE
[AIRFLOW-1299] Google Dataproc cluster creation operator should support imageVersion feature

### PR DESCRIPTION
**Pull request by yu-iskw**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- https://issues.apache.org/jira/browse/AIRFLOW-1299

### Description
I added `imageVersion` field to specify the Dataproc image version to `DataprocClusterCreateOperator`.

### CODE REVIEW

**Comment by yu-iskw** [comment]
@bolkedebruin @alexvanboxel @bodschut Can you review this PR, when you have time? Since this is my first PR for airflow, please let me know the manners if there is something wrong. Thanks!

**Comment by skudriashev** [comment]
@yu-iskw, please keep commit message within 50 characters. Thanks.

File: tests/contrib/operators/test_dataproc_operator.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from airflow.contrib.operators.dataproc_operator import DataprocClusterCreateOperator
+
+
+try:
```
Comment (skudriashev) [inline tests/contrib/operators/test_dataproc_operator.py]: I don't think you need this. Just use:

```python
import mock
```

Comment (yu-iskw) [reply tests/contrib/operators/test_dataproc_operator.py]: Thank you for the feedback. I will remove it.

**Comment by yu-iskw** [comment]
Thank you for merging it!

**Pull request closed without merging**